walkbox.go

```diff
@@ -175,11 +222,33 @@ func (wm *WalkBoxMatrix) EnableWalkBox(id int, enabled bool) {
 	}
 }
 
-// FindPath calculates and returns a path as a sequence of positions from the
+// WayPoint represents a point and its associated WalkBox.
+type WayPoint struct {
+	Walkbox  *WalkBox
+	Position Positionf
```

Comment:
Again, as this is part of the public interface of walkboxes, we should use integer positions here.